cordova-plugin-appcenter-crashes/plugin.xml

```diff
@@ -40,7 +40,7 @@
             <uses-permission android:name="android.permission.INTERNET" />
         </config-file>
 
-        <framework src="com.microsoft.appcenter:appcenter-crashes:2.3.0" />
+        <framework src="src/android/plugin.gradle" custom="true" type="gradleReference"/>
```

Comment:
How will this be reverted during release? I thought you wanted not to commit it: https://github.com/microsoft/appcenter-sdk-cordova/pull/117#issuecomment-539985585.